guides/development-workflow.md

```diff
@@ -461,26 +445,32 @@ story and the acceptance criteria that should be met.
 
 A good pull request should:
 
-- be created by the authoring developer
-- meet all of the acceptance criteria on the associated story
-- focus on the single problem at hand, not doing so will usually increase the
-  time taken to get it merged
-- include appropriate detail to assist the reviewer, this includes links to
-  relevant content and a duplication of anything of note in the commit
-  descriptions
-- update the [changelog](#tracking-changes) if it's a significant change
-
-```
-# Poor
-A new registration form and fix a bug with contacts
-
-# Great
-[123456789] New users can now register for an account
-
-- This has been rate limited, would like your feedback on the values I've chosen
-- Passwords are encrypted using BCrypt, so I've taken this opportunity to
-  increase the default rounds from 8 to 12
-```
+- describe the problem without assuming the reader has existing context
+- highlight the changes being made
+- describe and bring attention to particular difficulties or areas that you
+  particularly want the opinion of the reviewer
+- for user facing changes include screenshots of the before and after
+- the changes should be focused on meeting the acceptance criteria
+- focused on a single problem and preferring to split additional problems into
+  other pull requests, the smaller the pull request, the easier it is to be
+  reviewed and the quicker it is likely to be merged
+- updates the [changelog](#tracking-changes) if it's a significant change
+
+Real examples:
```

Comment:
Is this a good idea? I thought real, relatable examples would be more helpful than fictional ones. Plus it is nice to shine a light on good work. 

Sorry if you're not listed, I know I missed a few as I couldn't find recent work. Please do make suggestions and I'll get them in.